src/plugins/event-plugins/__tests__/XhrPlugin.test.ts

```diff
@@ -88,7 +91,7 @@ describe('XhrPlugin tests', () => {
         plugin.disable();
 
         // Assert
-        expect(record).toHaveBeenCalledTimes(1);
+        expect(record).toHaveBeenCalledTimes(2);
```

Comment:
question: Why do we expect this to now be called twice?

Reply:
Never mind, I see your comment in the other PR: https://github.com/aws-observability/aws-rum-web/pull/412/files#r1210735142

Reply:
Yeah sorry these PR's shouldn't have been combined. I wanted to keep the 2.0 changes together. Luckily, Quinn suggested to just set the target branch to `release/2.0.x`